helm-charts/charts/kube-starrocks/charts/starrocks/templates/starrockscluster.yaml

```diff
@@ -410,8 +412,9 @@ spec:
     support both resources and resource for backward compatibility
     */}}
     {{- if .Values.starrocksCnSpec.resources }}
-    {{- toYaml .Values.starrocksCnSpec.resources | nindent 4 }}
-    {{- else if .Values.starrocksCnSpec.resource }}
+    {{- include "starrockscluster.cn.resources" . | nindent 4 }}
+    {{- end }}
+    {{- if .Values.starrocksCnSpec.resource }}
```

Comment:
should merge the two `if` statements into a single if..end?

Reply:
committed